api/room_resource/schema.py

```diff
@@ -75,6 +75,48 @@ def mutate(self, info, **kwargs):
         return AssignResource(room_resource=room_resources)
 
 
+class UpdateAssignedResource(graphene.Mutation):
+    """
+        Update assigned resource in a room
+    """
+    class Arguments:
+        resource_id = graphene.Int(required=True)
+        room_id = graphene.Int(required=True)
+        quantity = graphene.Int(required=True)
+    room_resource = graphene.Field(RoomResource)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, resource_id, room_id, **kwargs):
```

Comment:
I think we could use `**kwargs` like this to reduce the number of arguments.
```suggestion
    def mutate(self, info, **kwargs):
```

Reply:
@verenceLola thanks for the review, however, I need to  pass only the quantity as key-value arguments to the update entity func which will not be possible with args and kwargs